src/components/CodeEditor/TestResultViewer.tsx

```diff
@@ -57,13 +57,16 @@ const TestResultTable = ({ editorOutput }) => {
       </div>
       <div className="h-48 overflow-auto">
         <table className="m-0 min-w-full table-auto border dark:border-slate-800/60">
-          <thead className="bg-gray-100 dark:bg-slate-800/60">
-            <tr>
+          <thead className="bg-white sticky top-0 dark:bg-slate-800/60">
```

Comment:
Previously the background color was `bg-gray-100`; why did we change the color to `bg-white`? Is there any specific reason?

Reply:
Sorry for that, its my mistake. i already changed this.